src/kixi/datastore/dynamodb.clj

```diff
@@ -258,7 +259,9 @@
   [conn table id-column id]
   (far/update-item conn table
                    {id-column id}
-                   {:update-map (map->update-map {::md/tombstone true})}))
+                   {:update-map
+                    (map->update-map {::md/tombstone true
+                                      ::md/tombstoned-at (t/timestamp)})}))
```

Comment:
this should come from the event or be made clear it is an aggregate time stamp, something we don't have a good mental model for yet

Reply:
Would you accept a given timestamp _or_ `(t/timestamp)` if not supplied?

Reply:
no, one or the other.